docker-compose.yaml

```diff
@@ -25,6 +25,7 @@ services:
     command: [serve]
     environment:
       - TZ=${TIMEZONE:-UTC}
+      - LOG_LEVEL=${LOG_LEVEL:-debug}
```

Comment:
we have a docker-compose.http-only.yaml as well

Reply:
Oh, yeah, is anyone using that? 🦅

Reply:
yes